src/Microsoft.Diagnostics.Monitoring.Options/DiagnosticPortOptions.cs

```diff
@@ -24,5 +24,10 @@ public class DiagnosticPortOptions
             ResourceType = typeof(OptionsDisplayStrings),
             Description = nameof(OptionsDisplayStrings.DisplayAttributeDescription_DiagnosticPortOptions_MaxConnections))]
         public int? MaxConnections { get; set; }
+
+        [Display(
+            ResourceType = typeof(OptionsDisplayStrings),
+            Description = nameof(OptionsDisplayStrings.DisplayAttributeDescription_DiagnosticPortOptions_ClearEndpointOnStartup))]
+        public bool? ClearEndpointOnStartup { get; set; }
```

Comment:
Should we have a `DefaultValue` here (following the pattern for `ConnectionMode`)?